tests/postgres_tests/test_aggregates.py

```diff
@@ -0,0 +1,252 @@
+import unittest
+
+from django.contrib.postgres.aggregates import (
+    ArrayAgg, BitAnd, BitOr, BoolAnd, BoolOr, Corr, CovarPop, RegrAvgX,
+    RegrAvgY, RegrCount, RegrIntercept, RegrR2, RegrSlope, RegrSXX, RegrSXY,
+    RegrSYY, StatFunc, StringAgg,
+)
+from django.db import connection
+from django.db.models.expressions import F, Value
+from django.test import TestCase
+from django.test.utils import Approximate
+
+from .models import GeneralTestModel, StatTestModel
+
+
+@unittest.skipUnless(connection.vendor == 'postgresql', 'PostgreSQL required')
+class TestGeneralAggregate(TestCase):
+    fixtures = ["aggregation_general.json"]
```

Comment:
:+1: on @charettes (thanks for the hint, didn't know this function exists :smile: )